everything.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
the current way this is written, I believe the output of lint / test is ignored, and the commit happens no matter what. You should add `set -e`:

`-e  Exit immediately if a command exits with a non-zero status.`